de_templates/test_config_ingest.yaml

```diff
@@ -0,0 +1,43 @@
+# `dataset_name` parameter is used to determine names of the following ADF resources:
+# - pipeline: Ingest_<dataset_name>
+# - dataset: ds_<dataset_name>
+# - linked service: ls_<dataset_name>
+dataset_name: AzureSql_Example
+pipeline_description: "Ingest from demo Azure SQL database using ingest config file."
+data_source_type: azure_sql
+
+key_vault_linked_service_name: ls_KeyVault
+data_source_password_key_vault_secret_name: sql-password
+
```

Comment:
Could we add in a parameter for `connection_string_variable_name`, which has the name used in variable group (in the existing example it's called `sql_connection`)? Just thinking this will be needed and will be unique for each data source.

Reply:
I can do it for the time being, but workload config file doesn't feel as a good name to provide password names or connection string variable names. I think it should be rather handled by some standard variables naming convention and passed properly within Terraform variables.

Reply:
done